### PR TITLE
Sanitise RegionFileCache and make cache size configurable.

### PR DESCRIPTION
**Pull request by cybertiger**

### CODE REVIEW

**Comment by aikar** [comment]
change looks fundamentally good, and i looked up why you did what you did, but can you add more to the patch commit message and be a bit more descriptive about what the problem is and how this fixes it (and why this is the best solution to use a LinkedHashMap), as i can see us accidently thinking this isn't doing much when we don't remember what exactly was wrong later down the road.

**Comment by cybertiger** [comment]
Updated the comment in the patch file, hopefully with enough info, let me know if I missed anything please.

File: Spigot-Server-Patches/0121-Sanitise-RegionFileCache-and-make-configurable.patch [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+diff --git a/src/main/java/net/minecraft/server/RegionFileCache.java b/src/main/java/net/minecraft/server/RegionFileCache.java
+index 5eb01c3..f1cc199 100644
+--- a/src/main/java/net/minecraft/server/RegionFileCache.java
++++ b/src/main/java/net/minecraft/server/RegionFileCache.java
+@@ -1,17 +1,18 @@
+ package net.minecraft.server;
+ 
++import com.destroystokyo.paper.PaperConfig;
+ import com.destroystokyo.paper.exception.ServerInternalException;
+-import com.google.common.collect.Maps;
```
Comment (kashike) [inline Spigot-Server-Patches/0121-Sanitise-RegionFileCache-and-make-configurable.patch]: Changing imports :(

**Comment by cybertiger** [comment]
Minimised diff with respect to imports and added //paper tags for imports.

**Pull request merged**